SIGs/SIG-dynamic-languages/proposal.md

```diff
@@ -0,0 +1,29 @@
+# Proposal to Create SIG-dynamic-languages
+
+This document is a proposal to create SIG-dynamic-languages as a formal Special Interest Group (SIG) under the auspices of the TSC of the Bytecode Alliance, as specified in the TSC’s charter.
+
+The purpose of this group is to make it easier for dynamic programming languages with non-trivial runtimes and interpreters to be compiled to Wasm.
+The group will attempt to achieve this by bringing together different interpreted language guest projects, identifying common foundational issues faced by those projects, and coordinating work on common tooling and solutions.
+
+The group will maintain lists of dynamic language guest projects, common foundational issues, implementation patterns, and common tools. The group will not be responsible for identified projects or tools but will act as a place for people to discover and collaborate on them and raise feedback to the Bytecode Alliance and its TSC.
+
+## Supporting Members
+
+The following individuals support the creation of $SIG_NAME:
```

Comment:
```suggestion
The following individuals support the creation of SIG-dynamic-laguages
```

Reply:
Thanks, I thought I'd changed this but I must have forgot to commit.